### PR TITLE
Add imagestream file for s390x and remove 3.2 version that is EOL

### PR DESCRIPTION
**Pull request by hhorak**
Similar PR to sclorg/mariadb-container#115

### CODE REVIEW

File: imagestreams/mongodb-rhel7-s390x.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    }
+  },
+  "spec": {
+    "tags": [
+      {
+        "name": "latest",
+        "annotations": {
+          "openshift.io/display-name": "MongoDB (Latest)",
+          "openshift.io/provider-display-name": "Red Hat, Inc.",
+          "description": "Provides a MongoDB database on RHEL 7. For more information about using this database image, including OpenShift considerations, see https://github.com/sclorg/mongodb-container/tree/master/3.4/README.md.\n\nWARNING: By selecting this tag, your application will automatically update to use the latest version of MongoDB available on OpenShift, including major version updates.",
```
Comment (phracek) [inline imagestreams/mongodb-rhel7-s390x.json]:
```suggestion
          "description": "Provides a MongoDB database on RHEL 7. For more information about using this database image, including OpenShift considerations, see https://github.com/sclorg/mongodb-container/tree/master/3.6/README.md.\n\nWARNING: By selecting this tag, your application will automatically update to use the latest version of MongoDB available on OpenShift, including major version updates.",
```

**Review by phracek — CHANGES_REQUESTED** [review]
Just one small fix. Otherwise LGTM.

File: imagestreams/mongodb-rhel7-s390x.json (hunk 1 above)
Comment (pkubatrh) [reply imagestreams/mongodb-rhel7-s390x.json]: Hmm would love to commit this but "You do not have permission to push to this repository."

@hhorak Do you perhaps have "Allow edits from maintainers." unchecked?

**Comment by hhorak** [comment]
@pkubatrh yes, my fault, but I've just committed the suggestion.

**Pull request merged**